sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -81,6 +81,7 @@ public class Configs {
     private static final Duration CONNECTION_ACQUIRE_TIMEOUT = Duration.ofSeconds(45);
     private static final int REACTOR_NETTY_MAX_CONNECTION_POOL_SIZE = 1000;
     private static final String REACTOR_NETTY_CONNECTION_POOL_NAME = "reactor-netty-connection-pool";
+    private static final int GATEWAY_RESPONSE_TIMEOUT = 65;
```

Comment:
I think in java we have so far used the 60s DEFAULT_HTTP_RESPONSE_TIMEOUT_IN_SECONDS - why change it now. I don't think I have ever seen a request succeeding from Gateway in over 60 seconds (but less than 65 seconds).

Reply:
.Net had a timeout of 65 seconds, but as you mentioned in Java SDK we have been using 60 secs as the default HTTP timeout. I have removed this from the code.